stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/plugin/logic/channel/internal/ChannelLogic.kt

```diff
@@ -124,6 +124,13 @@ internal class ChannelLogic(
     private val attachmentUrlValidator: AttachmentUrlValidator = AttachmentUrlValidator(),
 ) : QueryChannelListener {
 
+    private val messageIdsBellowGap = mutableListOf<Long>()
+    private val messageIdsAboveGap = mutableListOf<Long>()
```

Comment:
Can we rename it in respect of the historical axis?

Reply:
I thought about naming then older and newer. But this got confusing. This part of the logic is very oriented by the direction and location of the messages and not so much about history. I would get confused using names related to historical axis.